Lib/test/test_asyncio/test_streams.py

```diff
@@ -1095,6 +1096,39 @@ async def inner(httpd):
 
         self.assertEqual(messages, [])
 
+    def test_loop_is_closed_resource_warnings(self):
+        async def inner(httpd):
+            rd, wr = await asyncio.open_connection(*httpd.address)
+
+            wr.write(b'GET / HTTP/1.0\r\n\r\n')
+            data = await rd.readline()
+            self.assertEqual(data, b'HTTP/1.0 200 OK\r\n')
+            data = await rd.read()
+            self.assertTrue(data.endswith(b'\r\n\r\nTest message'))
+            self.loop.stop()
+            wr.close()
+            while not wr._loop.is_closed():
```

Comment:
You can use `self.loop` here, right?
```suggestion
            while not self.loop.is_closed():
```